manifests/w/Wildy/bobonslaught/faf57/Wildy.bobonslaught.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Wildy.bobonslaught
+PackageVersion: faf57
+PackageLocale: en-US
+Publisher: Wildy
+PackageName: (NEW UPDATE) literally every fnf mod ever (Vs Bob)
+License: Apache 3
+ShortDescription: Uh Oh, the man Bob is angry and you made them angry and they are so angry man, what did you do to make them angry? Honestly dont know Im just a game description.

```

Comment:
Please update this section with better data - include the PublisherUrl, correct the license, and add a better answer for the other 2 lines.